src/Tizen.NUI/src/public/BaseHandle.cs

```diff
@@ -405,6 +405,11 @@ public bool HasBody()
                 return false;
             }
 
+            if (swigCPtrCopy.Handle == IntPtr.Zero)
```

Comment:
I think it would be better that this codes be moved line 403 where is above "if(disposed == true)"
also could you add some debugging log like this?

            global::System.Diagnostics.StackTrace t = new global::System.Diagnostics.StackTrace();
            Tizen.Log.Error("NUI", t);
            Tizen.Log.Error("NUI", "[ERROR] the native handle is invalid!");

Reply:
fixed